app/controllers/posts_controller.rb

```diff
@@ -1,29 +1,41 @@
 class PostsController < ApplicationController
```

Comment:
- [ ] It would be nice to add the` load_and_authorize_resource`  to your Post Controller  as it is  part of the CanCanCan gem and `With load_and_authorize_resource`, the instance variables like `@user`,` @post,` and @post_item are automatically loaded and authorized. You no longer need to manually load and authorize these resources 👍🏿

Reply:
- [ ] Great job adding the `load_and_authorize_resource` to your Post Controller as requested. However, you have introduced an error as you have not properly configured access rights. Remember that the purpose of this milestone is so that Only a User that created a post can delete it or a User that has Adminstrator rights. Currently, A logged in user is unable to create their own Post. Kindly fix this as this bug has been introduced in recent commits

Before New Commits
[cinnamon-2023-11-18T155705+0200.webm](https://github.com/Graciano1997/technology_blog_app/assets/81932478/6c5e3008-0be3-4acf-a7ea-133e0bc0862d)


After new commits
[cinnamon-2023-11-18T155846+0200.webm](https://github.com/Graciano1997/technology_blog_app/assets/81932478/6e7bc54a-4ade-4175-b5cc-dc68520aec5c)